src/app/components/work-item-detail/work-item-detail.component.less

```diff
@@ -279,45 +279,45 @@
       max-height: 150px;
       overflow-y: scroll;
       border-bottom: 1px solid;
-      border-bottom-color: $color-pf-black-500;
+      border-bottom-color: @color-pf-black-500;
     }
   }
 }
-@media (max-width: $grid-float-breakpoint){
+@media (max-width: @grid-float-breakpoint){
   /*Toggle the details panel's width and left margin*/
   .detail-container {
     .top-bar {
       padding: 10px 20px;
-      color: $color-pf-white;
-      background-color: $color-pf-black;
+      color: @color-pf-white;
+      background-color: @color-pf-black;
     }
     .detail-type-dropdown { position: relative; }
     .wi-state-btn{ padding-top: 0; }
   }
   .detail-panel {
     top: 0;
-    width: $width100;
+    width: 100%;
     padding: 0;
   }
   .user-search {
     position: fixed;
     top: 0;
     left: 0;
     z-index: 9;
-    width: $width100;
+    width: 100%;
     height: 100%;
     background-color: rgba(0, 0, 0, 0.6);
     .list-container {
       top: 144px;
       left: 50%;
       width: 80%;
-      @include transform(translateX(-50%));
+      .transform(translateX(-50%));
       .user-list { max-height: 250px; }
     }
   }
 }
-.user-avatar { @include user-avatar; }
+.user-avatar { .user-avatar; }
 .detail-action-btn {
-  @include detail-action-btn;
+  .detail-action-btn;
```

Comment:
at some places this mixin has additional css... so kept it as mixin for now...later it should be refactored